en/docs/consume/invoke-apis/invoke-apis-using-tools/invoke-an-api-using-a-soap-client.md

```diff
@@ -43,9 +55,10 @@ Let's invoke the `PhoneVerification` API using a SOAP client.
         ![]({{base_path}}/assets/attachments/103332601/103332589.png)
 
     Note the result on the right-hand side panel. As you gave a dummy phone number in this example, you get the result as invalid.
-    ![]({{base_path}}/assets/attachments/103332601/103332590.png)
+    
+![]({{base_path}}/assets/attachments/103332601/103332590.png)
```

Comment:
If you come across images or attachments with numbers like this please rename them using a meaningful name.

Reply:
Ack